### PR TITLE
Add release helper and structure workflow files

### PR DESCRIPTION
**Pull request by pat-s**
So far everything was done in one workflow file. Splitting steps into dedicated files like we do in other repos already helps for overview and future modifications.

Also added tags to images.

### CODE REVIEW

**Comment by anbraten** [comment]
We need to wait for https://github.com/woodpecker-ci/woodpecker/pull/2562 to be able to update the config path.

File: .woodpecker/publish-release.yml [hunk 1]
```diff
@@ -2,33 +2,8 @@ variables:
   - &golang 'golang:1.21'
```
Comment (anbraten) [inline .woodpecker/publish-release.yml:2]: should depend on test

File: .woodpecker/release-helper.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+steps:
+  release-helper:
+    image: woodpeckerci/plugin-ready-release-go:0.6.1
+    pull: true
```
Comment (anbraten) [inline .woodpecker/release-helper.yml:4]:
```suggestion
```

**Review by anbraten — APPROVED** [review]

**Comment by pat-s** [comment]
@anbraten A GH_TOKEN is missing in the WP repo

**Pull request merged**